### PR TITLE
Aravindraveendran7 patch 2

### PR DESCRIPTION
**Pull request by aravindraveendran7**
Added the following changes in the profile

- Added Avatar
- Added socials icon
- Added more relevant tags

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2639"></a>

### CODE REVIEW

**Review by ChinmayMhatre — APPROVED** [review]
Looks good!

File: data/aravindraveendran7.json [hunk 1]
```diff
@@ -2,8 +2,14 @@
...
   "type": "personal",
-  "bio": "Test Automation Engineer **Devops Enthusiast**. Stock market enthusiast|Swing Trader|Learning Derviative Trading",
-  "avatar": "https://github.com/aravindraveendran7",
+  "bio": "Test Automation Engineer |DevOps Enthusiast|Stock market enthusiast|Swing Trader|Learning Derviative Trading",
+  "tags": ["Open Source", "DevOps", "Selenium", "Java", "Rest API","Docker","QA Automation"],
+  "avatar": "https://github.com/aravindraveendran7.png",
+  "socials": [
+    { "icon": "FaTwitter",  "url": "https://twitter.com/AravindRaveend6" },
+    { "icon": "FaGithub",   "url": "https://www.linkedin.com/in/aravind-raveendran700" },
+    { "icon": "FaLinkedin", "url": "https://pradumnasaraf.dev" }
```
Comment (Panquesito7) [inline data/aravindraveendran7.json:11]: You might want to update this link here.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/aravindraveendran7.json (hunk 1 above)
Comment (aravindraveendran7) [reply data/aravindraveendran7.json:11]: @Panquesito7  sorry for the typo there and thank you for pointing it out 👍 . Raised a new PR now.

Comment (eddiejaoude) [reply data/aravindraveendran7.json:11]: Please close this PR if you no longer want it

**Comment by aravindraveendran7** [comment]
closing PR #2639

File: data/aravindraveendran7.json (hunk 1 above)
Comment (aravindraveendran7) [reply data/aravindraveendran7.json:11]: Done @eddiejaoude

**Pull request closed without merging**